product_development.md

```diff
@@ -0,0 +1,96 @@
+
+### Platform announcement Slack channel
+
+Product, Design, Engineering, and Content team members post every change to go live on the platform into the Slack channel [#platform-announcements](https://voiio.slack.com/archives/C02BXK6R16V). This channel serves as a changelog for all employees.
+
+## Meetings & Collaboration
+
+### Agile principles
+
+* Make work visible
+* Meaningful & actionable conversations
+* Decisions are made, where the work is done
+* Regularly reflect on how we work together
+
+### Stakeholder Meetings
+
+Stakeholder meetings are held weekly to discuss the product roadmap, upcoming features, and any blockers that need to be resolved or questions that need to be answered. It is the best path to discuss new ideas and to align on the product's direction. The product manager will relate work to the OKR progress and stakeholders will come prepared with issues/ideas and data, adding their topics to this document: [Stakeholder Meeting Platform](https://docs.google.com/document/d/1uhX68whx9T56unUlEUdyjUdq_dOJpHwO7Ycv892hqA8/edit)
+
+The meeting is led by the Product Manager and attended by key stakeholders and an engineering lead.
+
+### Refinement Meetings
+
+Refinement meetings are held weekly and are the handover point from the Product Team to the Engineering Team. The product manager presents problems, opportunities, and priorities to the Engineering Team. Engineers ask questions and either pull the issue into the backlog or push it back to the product team for further refinement.
+
+The meeting is led by the product manager and attended by the product and engineering team.
+
+### Standups
+
+Standups are held daily and are the heartbeat of the team (Engineering, Content, Product). They are used to update the team on the progress of ongoing tasks and blockers. Standups are not for problem-solving, but for identifying issues that need to be solved later during the day.
+
+Standups are led by the respective Team Lead going through the Kanban board and attended by the entire team and, optionally, by people who have been closely working with the respective team.
+
+### Retrospectives
+
+Retrospectives are held every month with the Engineering, Product and Design Team. Stakeholders may join optionally or can be invited if they have been highly involved with a project.
+
+### Review
+
+We have a continuous development and deployment process and therefore no regular review meeting. However, reviews are an integral part of the development process. Engineers are responsible to include key stakeholders or product into the solution building and review process.
+
+We strictly prioritize closing open work over everything else. Therefore, any party that has been requested to review a feature is expected to make time within a business day.
+
+Client-facing features may require a scheduled deployment coordinated with our clients by CSM.
```

Comment:
May or may not - do we need to elaborate on this?

Reply:
I honestly don't know yet. Let's see.